Library/Formula/cucumber-cpp.rb

```diff
@@ -0,0 +1,24 @@
+class CucumberCpp < Formula
+  desc "Support for writing Cucumber step definitions in C++"
+  homepage "https://cucumber.io"
+  url "https://github.com/cucumber/cucumber-cpp/archive/1e633c6b59c75b959b480dcc4e40123e7fc59032.tar.gz"
+  version "0.3"
+  sha256 "1e211db8b8c230f291c8774d1c2e2bbbcf264d92016fd6081557425e43569c19"
+
+  depends_on "cmake" => :build
+  depends_on "ruby"
```

Comment:
As I said in a previous comment:

> In fact this would be needed just for (currently empty) testing purpose. Once a program is compiled with this library, it'll start listening on the port 3902, we need to execute the cucumber gem against it.